clang/docs/checkedc/3C/INSTALL.md

```diff
@@ -84,23 +100,3 @@ https://developer.apple.com/download/more/, log in with your Apple
 Developer ID, and download "Command Line Tools for Xcode 11.3.1".
 Then, enable these by invoking (from the command line), `sudo
 xcode-select -r`.
-
-## CR/LF issues
```

Comment:
Why did this get removed? Is it because this relates to building/maintaining? Because it's no longer an issue?

Reply:
The problem has been fixed.  This was noted in the linked issue, but I had forgotten that I need to close that issue when this PR is merged, so I'm glad you brought it up.